Makefile

```diff
@@ -51,13 +51,21 @@ tools: $(GOLICENSES) $(MULTIMOD) $(GOLANGCI_LINT) $(DBOTCONF) $(OFFSETGEN)
 
 ALL_GO_MODS := $(shell find . -type f -name 'go.mod' ! -path '$(TOOLS_MOD_DIR)/*' ! -path './LICENSES/*' | sort)
 GO_MODS_TO_TEST := $(ALL_GO_MODS:%=test/%)
+GO_MODS_TO_EBPF_TEST := $(ALL_GO_MODS:%=test_ebpf/%)
```

Comment:
Sorry for the possibly naive question, but where are these files? I'm not seeing any `test_ebpf` directories?

Reply:
My intention here was to create a separate make target for running unit tests that require sudo (loading maps and programs).
Examples of these tests are in `internal/pkg/instrumentation/probe/sampling/sampling_linux_test.go`.

Since we have targets for `test/<mod_name>`, I figured these would be the equivalent of that to avoid conflicting target names.